archey/entries/cpu.py

```diff
@@ -8,9 +8,21 @@
 
 from archey.distributions import Distributions
 from archey.entry import Entry
+from archey.configuration import Configuration
 
 
 class CPU(Entry):
+
```

Comment:
Instead of this in every entries, I propose that a new protected `_ICON` is added to them and when the configuration enables it, the value is retrieved and prepended to `_PRETTY_NAME` (in `Entry` base definition) as you suggested.